fission-web-server/library/Fission/Web/Server/Handler/App/Index.hs

```diff
@@ -23,10 +24,10 @@ index Authorization {about = Entity userId _} = runDB do
   let normalized = (\(k, v) -> (fromIntegral (fromSqlKey k), v)) <$> appXDomains
   return (Map.fromList normalized)
   where
-    findDomains :: App.Domain.Retriever m => Entity App -> m (AppId, [URL])
-    findDomains (Entity appId _) = do
+    findDomains :: App.Domain.Retriever m => Entity Models.App -> m (AppId, Core.App)
```

Comment:
See comment above about `App.Payload` vs `App`; it should also make this read better 👍